### PR TITLE
Tailwind theme

### PR DESCRIPTION
**Pull request by lindsaym-fa**
Adds a theme to match the vibes of https://tailwindui.com/

### CODE REVIEW

File: docs/pages/experimental/kitchen-sink.md [hunk 1]
```diff
@@ -1265,6 +1284,16 @@ toc: false
...
+  .message-composer wa-card::part(header) {
+    border-start-start-radius: calc(var(--border-radius) - var(--border-width));
+    border-start-end-radius: calc(var(--border-radius) - var(--border-width));
+  }
+
+  .message-composer wa-card::part(footer) {
+    border-end-start-radius: calc(var(--border-radius) - var(--border-width));
+    border-end-end-radius: calc(var(--border-radius) - var(--border-width));
+  }
+
```
Comment (lindsaym-fa) [inline docs/pages/experimental/kitchen-sink.md:1296]: Moved these styles out of playful.css and into the general kitchen sink styles.

File: docs/pages/experimental/kitchen-sink.md [hunk 2]
```diff
@@ -1321,6 +1350,10 @@ toc: false
     --size: var(--wa-font-size-2xl);
   }
 
+  .support-table wa-card > * {
+    border-radius: calc(var(--border-radius) - var(--border-width));
+  }
```
Comment (lindsaym-fa) [inline docs/pages/experimental/kitchen-sink.md:1355]: Moved these styles out of playful.css and into the general kitchen sink styles.

File: src/themes/headwind.css [hunk 3]
```diff
@@ -0,0 +1,563 @@
...
+}
+
+.project-header {
+  background: var(--wa-color-surface-default);
+}
+
+.hero {
+  --hero-background-color: var(--wa-color-surface-default);
+  --hero-lines-color: color-mix(in oklab, var(--wa-color-neutral-fill-highlight), transparent 30%);
+  background: linear-gradient(to top, var(--wa-color-surface-lowered), transparent 40%),
```
Comment (frrrances) [inline src/themes/headwind.css:447]: Wow, that is quite a background! Looks awesome!

**Review by frrrances — COMMENTED** [review]
Looks great, @lindsaym-fa! Code looks spotless - the only thing I found that you might want to adjust is the neutral button hover and active states. Since they're black, there's no interaction feedback that let's you know either of those states are active.

**Comment by lindsaym-fa** [comment]
> Looks great, @lindsaym-fa! Code looks spotless - the only thing I found that you might want to adjust is the neutral button hover and active states. Since they're black, there's no interaction feedback that let's you know either of those states are active.

Great call, @frrrances! Adjusted. ✔️

**Pull request merged**